static/bootstrapalert.js

```diff
@@ -52,6 +52,49 @@ const bootstrap5Alert = ({ message, title, cb }) => {
   }
 }
 
+const bootstrap5Confirm = ({ message, title, cb }) => {
```

Comment:
This function is more or less the same with the bootstrap5Alert one. Please merge this one to the bootstra5Alert one. Pass extra parameters if needed if you want to differentiate between them. Also in the latest version of the project I have changed the bootstrap5Alert a bit, please take this also into account.